herbert_core/utilities/classes/@unique_objects_container/unique_objects_container.m

```diff
@@ -420,6 +420,21 @@
             % - hash : the resulting has, a row vector of uint8's
             %
             persistent Engine;
+            %{
+            % monitor for use of hashing. As this issue may continue, leaving it in the code
+            persistent count;
+            if nargin>2
+                count=0;
+                hash = []; % unused null value for this case
+                return;
+            end
+            if isempty(count)
+                count=0;
+            end
+            count=count+1;
+            count
+            disp(class(obj));
+            %}
```

Comment:
Could put this in an `if exist('reset_count', 'var')` block so if you want this info, then you set `reset_count == false` or probably better add a `debug` hidden property to the class which handles this

Reply:
may well do this in due time